src/libraries/shims/ApiCompatBaseline.PreviousNetCoreApp.txt

```diff
@@ -1,8 +1,18 @@
-Compat issues with assembly System.Diagnostics.Process:
+Compat issues with assembly netstandard:
+CannotChangeAttribute : Attribute 'System.ComponentModel.DesignerAttribute' on 'System.ComponentModel.IComponent' changed from '[DesignerAttribute("System.ComponentModel.Design.ComponentDesigner, System.Design, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a")]' in the contract to '[DesignerAttribute("System.ComponentModel.Design.ComponentDesigner, System.Design, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a")]' in the implementation.
```

Comment:
I see that we're catching this now as a diff with past release.  Does API compat flag the original issue as a diff with netfx?  If so are there any other similar diffs?

Reply:
No it didn't catch it. I don't know why. Based on reading these errors, it seems like APICompat is not catching the right "Diff" the diff is in the `DesignerAttribute` that specifies a `ComponentDocumentDesigner`.  

This is the only diff I get on my local build and on CI.